.circleci/config.yml

```diff
@@ -1932,7 +1932,7 @@ jobs:
             cd /; GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" \
               git clone  --recurse-submodule ssh://git@github.com/swirlds/swirlds-platform.git
             cd /swirlds-platform; git submodule update --init --recursive --checkout
-            cd /swirlds-platform/regression;   git checkout 00438-D-migration-test-for-mainnet;
+            cd /swirlds-platform/regression;   git checkout 334-D-Add-services-config;
```

Comment:
Just a thought. Should we add `git pull` after checking out 334-D-Add-services-config branch ?? Not sure if that will make any difference

Reply:
I don't think we need that. Since it's in a new env and it should be the latest of the branch.